misc/ls-extensions/modules/json-to-record-converter/src/main/java/io/ballerina/converters/JsonToRecordConverter.java

```diff
@@ -124,7 +125,7 @@ public static JsonToRecordResponse convert(String jsonString, String recordName,
         } else {
             // Sets generated type definition code block
             NodeList<ModuleMemberDeclarationNode> moduleMembers = AbstractNodeFactory.createNodeList(
-                    typeDefinitionNodeList.toArray(new TypeDefinitionNode[0]));
+                    typeDefinitionNodes.values().toArray(new TypeDefinitionNode[0]));
```

Comment:
Any specific reason to use the `toArray`? Can't we use stream API and map to a list and provide that to the `createNodeList`?

Reply:
Addressed in https://github.com/ballerina-platform/ballerina-lang/pull/34525/commits/9129a28d18a2ad3bae9e19515f274b8ae939c033